### ISSUE TITLE
Header absolute/fixed value with transparent background

### ISSUE THREAD

**Issue by IbraheemAlSaady** [GeekyAnts/NativeBase#1981]
I'm trying to make the header to be transparent. I've tried styling it to `absolute` but its not in the screen (hidden) if I do that. also if I do `backgroundColor: 'transparent'` it doesn't work as well. 

![image](https://user-images.githubusercontent.com/14288490/41194515-01709276-6c25-11e8-84a8-21148253e5f1.png)

I want the `Header` component to be on top of the view that has the gradient. I want to achieve something like this.

![image](https://user-images.githubusercontent.com/14288490/41194538-5237ec86-6c25-11e8-9bed-f44df9be5cf3.png)

Is it possible with Native Base header component or I have to create my own?

**Comment by QinGuan2004** [comment]
To make header transparent you should do 
```class App extends Component {
  static navigationOptions = {
    headerStyle: {
      backgroundColor: 'transparent'
    }
  }
  render() {
    return (
      <YourViewHere/>
    )
  }
}
```

You did ```backgroundColor: { 'transparent' } ``` instead with {}
Also keep note that the white _*may*_ be your actual background color, so you may mistake your View background color for that

**Comment by IbraheemAlSaady** [comment]
@QinGuan2004 Yeah sorry for the `{}`, I updated my question.
I'm using the `Header` component from Native Base, looks like this.

```js
<Header noShadow style={{ backgroundColor: 'red' }}>
   <Left>
       <Button transparent>
           <Icon name="arrow-back" />
       </Button>
   </Left>
</Header>
```
You can change the style of the `Header` (backgroundColor, etc...), but what I want is for the header to be on top of the element that has the gradient color.

I tried your solution, but unfortunately, it didn't work.

**Comment by akhil-geekyants** [comment]
@IbraheemAlSaady try wrapping the screen with `ImageBackground`. 

Sample code
```
import React, { Component } from 'react';
import { ImageBackground, Dimensions } from "react-native";
import { Container, Header, Left, Body, Right, Button, Icon, Content, Text } from 'native-base';

export default class App extends Component {

  render() {

    let { height, width } = Dimensions.get('window');

    return (
      <ImageBackground source={require("./background.png")} style={{ width, height }}>
        <Container>
          <Header style={{ backgroundColor: "transparent" }} iosBarStyle={"light-content"} noShadow>
            <Left>
              <Button transparent>
                <Icon name='arrow-back' style={{ color: "white" }} />
              </Button>
            </Left>
            <Body />
            <Right />
          </Header>
          <Content>
            <Text style={{ color: "white", padding: 20 }}>Hi there!</Text>
          </Content>
        </Container>
      </ImageBackground>
    );
  }
}
```

Screenshot
![simulator screen shot - iphone 6 - 2018-06-11 at 15 30 43](https://user-images.githubusercontent.com/27768184/41225617-d32e1924-6d8c-11e8-9d82-69d98f881c10.png)

**Comment by SupriyaKalghatgi** [comment]
`noShadow` does not give the desired output for iOS

Lets have `transparent` as prop for Header

**Comment by SupriyaKalghatgi** [comment]
Fixed with [v2.5.2](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.5.2)

**Comment by IbraheemAlSaady** [comment]
@SupriyaKalghatgi, v2.5.2 did the trick. thanks

**Comment by kenny26** [comment]
I'm using the latest version 2.5.2, and I can't even reproduce what @akhil-geekyants have on the screen...

Here is my Screen
```
import React, { Component } from 'react';
import { ImageBackground, Dimensions } from "react-native";
import { Container, Header, Left, Body, Right, Button, Icon, Content, Text } from 'native-base';

export default class App extends Component {

  render() {
    let { height, width } = Dimensions.get('window');

    return (
      <ImageBackground
        source={{uri: "https://placeimg.com/1000/2160/tech"}}
        style={{ width, height }}
      >
        <Container>
          <Header
            style={{ backgroundColor: "transparent" }}
            iosBarStyle={"light-content"}
            androidStatusBarColor="rgba(0,0,0,0.5)"
            transparent
            noShadow
          >
            <Left>
              <Button transparent>
                <Icon name='arrow-back' style={{ color: "black" }} />
              </Button>
            </Left>
            <Body />
            <Right />
          </Header>
          <Content>
            <Text style={{ color: "black", padding: 20 }}>Hi there!</Text>
          </Content>
        </Container>
      </ImageBackground>
    );
  }
}
```

And here's the result I got
![screenshot_1529283656](https://user-images.githubusercontent.com/13347016/41514049-fa82a71a-72cd-11e8-907e-149546e219cc.png)

While the header is transparent, I can't get the status bar to become transparent... Any ideas?

**Comment by IbraheemAlSaady** [comment]
@kenny26  You can do something like this on android

```js
import { StatusBar } from 'react-native'

// inside your component class
componentWillMount() {
    if (Platform.OS === 'android') {
      StatusBar.setTranslucent(true);
    }
}

// you can put this inside your render method
<StatusBar backgroundColor="transparent" barStyle="light-content" />
```

**Comment by SupriyaKalghatgi** [comment]
Fixed with [2.6.1](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.6.1)

**Comment by iamadityaaz** [comment]
I done this using `style={{ backgroundColor: null }}`
